src/t2iapi/combined/combined_requests.proto

```diff
@@ -29,3 +29,29 @@ message CreateContextStateWithAssociationAndSetOperatingModeRequest {
                                             // shall be set
     t2iapi.operation.OperatingMode operating_mode = 4;  // the requested @OperatingMode value of the operation state
 }
+
+/*
+Request to create a new or adapt an existing component activation state to the given ComponentActivation value and
+set the requested @OperatingMode for the referenced operation.
+ */
```

Comment:
adapt to something like:
/*
Request to set the ActivationState of the component or metric with the given handle to the requested ComponentActivation value and to set the OperatingMode of the operation with the given handle to the requested OperatingMode value.
*/